mne/io/eeglab/eeglab.py

```diff
@@ -347,8 +347,8 @@ def __init__(self, input_fname, montage, eog=(), event_id=None,
         info['chs'].append(stim_chan)
         info._update_redundant()
 
-        events = _read_eeglab_events(eeg, event_id=event_id,
-                                     event_id_func=event_id_func)
+        events = read_events_eeglab(eeg, event_id=event_id,
+                                    event_id_func=event_id_func)
```

Comment:
>if you don't pass event_id to read_raw_eeglab and there is no warning, some events are still mapped to integers?

The mapping is predictable and documented. We discussed this when creating the function. There are event_id_func and event_id, both of which are documented and user-accessible. If an event is not caught by either, a warning is raised and the user is told to use event_id_func or event_id. Makes sense?